src/components/storyCard/index.tsx

```diff
@@ -0,0 +1,124 @@
+    owners: {
+      backend?: Owner,
+      frontend?: Owner,
+      feature?: Owner
+    }
+  }
+}
+
+const Task: FC<TaskProps> = ({ title, status }) => (
+  <div className={styles.subtask}>
+    <span className={styles.subtask_title}>{title}</span>
+    <span className={styles.subtask_status}>{status}</span>
+  </div>
+);
+
+const StoryCard: FC<Props> = ({ data }) => {
+  const {
+    title, description, subtasks, status, owners,
+  } = data;
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.task_info}>
+        <div className={styles.title}>
+          {title}
+        </div>
+        <div className={styles.description}>
+          {description}
+        </div>
+        <div className={styles.subtasks}>
+          {subtasks.map((task) => (<Task title={task.title} status={task.status} />))}
+        </div>
+        <div className={styles.date_info}>
+          <div className={styles.task_started}>
+            Started 12 days ago
+          </div>
+          <div className={styles.task_due}>
+            <img src="/calendar-icon.png" alt="calendar-icon" height="25" />
+            Due Date
+          </div>
+        </div>
+      </div>
+      <div className={styles.task_status}>
+        <div className={styles.status_info}>
+          Status:
+          {status === 'Active'
+            ? <span className={styles.subtask_status}>{status}</span>
+            : <img src="/lock-solid.svg" alt="lock-icon" height="30" />}
+        </div>
+        <div>
+          <div className={styles.owner_info}>
+            <p>Feature Owner:</p>
+            {owners.feature
+              ? (
+                <img
+                  className={styles.owner_image}
+                  src={owners.feature.img}
+                  alt={owners.feature.name}
+                />
+              )
+              : (
+                <div className={styles.to_be_decided}>
+                  <p>TBD</p>
+                </div>
+              )}
+          </div>
+          <div className={styles.owner_info}>
+            <p>Backend Engineer:</p>
+            {owners.backend
+              ? (
+                <img
+                  className={styles.owner_image}
+                  src={owners.backend.img}
+                  alt={owners.backend.name}
+                />
+              )
+              : (
+                <div className={styles.to_be_decided}>
+                  <p>TBD</p>
+                </div>
+              )}
+          </div>
+          <div className={styles.owner_info}>
+            <p>Frontend Engineer:</p>
+            {owners.frontend
+              ? (
+                <img
+                  className={styles.owner_image}
+                  src={owners.frontend.img}
+                  alt={owners.frontend.name}
+                />
+              )
+              : (
+                <div className={styles.to_be_decided}>
+                  <p>TBD</p>
+                </div>
+              )}
+          </div>
```

Comment:
the `owner_info` div has almost a similar structure and it's repeating 3 times.
I think we can create a function `renderOwnerInfo` and pass the necessary data and that function will return this `div structure`.
What say?

Reply:
Yeah, makes sense. Extracted the common parts and converted it to a function. Thanks.